src/_includes/graphql/customer-orders-output.md

```diff
@@ -107,26 +75,17 @@ Attribute | Data type | Description
 --- | --- | ---
 `comments` | [[SalesCommentItem](#SalesCommentItem)] | Comments on the credit memo
```

Comment:
It's not related to the current PR, but just to mention that we have some changes introduced in this schema 

```suggestion
`comments` | [[CommentItem](#SalesCommentItem)] | Comments on the credit memo
```

https://github.com/magento/magento2/blob/20b7e0d503f1b50fe2ca7385cf41657501b47398/app/code/Magento/SalesGraphQl/etc/schema.graphqls#L56